New package: reoMS v0.0.1

JuliaRegistrator opened the pull request:
- Registering package: reoMS
- Repository: https://github.com/yanjer/reoMS.jl
- Created by: @yanjer
- Version: v0.0.1
- Commit: d07d9633272ff5737f8b2c1c1a14fe8df4f8934c
- Git reference: HEAD

goerz commented:
Something like `REOMolecularS…` (whatever “S” stands for) might be a suitable package name. Long and descriptive package names are strongly encouraged in Julia.

JuliaHubRegistrator committed:
New package: reoMS v0.0.1

UUID: 8f17fb8d-1e49-4323-bc6d-d679a20c8a2d
Repo: https://github.com/yanjer/reoMS.jl.git
Tree: 5cdbdcc9a9e833a4930861885348796e37781828

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8

goerz commented:
https://github.com/JuliaRegistries/General/pull/116320
